aspnet-core/xml/ns-Microsoft.AspNetCore.Authorization.xml

```diff
@@ -1,6 +1,5 @@
 <Namespace Name="Microsoft.AspNetCore.Authorization">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains types that enable support for authorization.</summary>
```

Comment:
This namespace is pretty important. It should have some more information. See some of the longer summaries at https://docs.microsoft.com/en-us/dotnet/api/?view=netcore-3.1

Also add a link in the remarks:

`<remarks>For more information, see <see href="https://docs.microsoft.com/aspnet/core/security/authorization/introduction">Introduction to authorization in ASP.NET Core</see>.</remarks>`